src/pages/Recovery/Recovery.tsx

```diff
@@ -60,7 +60,7 @@ const Recovery = () => {
         console.log();
```

Comment:
@quyn2904 add prevent console.log when commit

Reply:
I have added this rules at the latest 3 commits of this issue